src/main/java/org/openstreetmap/atlas/geography/Polygon.java

```diff
@@ -190,6 +191,12 @@ public boolean fullyGeometricallyEncloses(final Location location)
         }
     }
 
+    @Override
+    public boolean fullyGeometricallyEncloses(final MultiPolygon multiPolygon)
+    {
+        return multiPolygon.outers().stream().allMatch(this::fullyGeometricallyEncloses);
```

Comment:
What if this polygon entirely falls within an inner polygon of the multipolygon passed in? I believe this is not a covered use case here? If not, mind adding a test for that specific corner case?

Reply:
For a polygon to geometrically enclose a multi-polygon, it must geometrically enclose all the outer polygons of the multi polygon. If the polygon lies within the inner of the passed in multi-polygon then, the polygon does not geometrically enclose the passed in multi-polygon.

Reply:
I added a test case to display the interaction I believe to be desirable based on what I understand of the method.